Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/Systems/MovementBadgesSystem.cs

```diff
@@ -37,31 +39,56 @@ public partial class BadgesHeightReachedSystem : BaseUnityLoopSystem
         private float totalElevationGain;
         private IWeb3Identity? currentIdentity;
 
-        public BadgesHeightReachedSystem(
+        public MovementBadgesSystem(
             World world,
             IAnalyticsController analytics,
             IRealmData realmData,
             in Entity playerEntity,
             IWeb3IdentityCache identityCache,
-            IDebugContainerBuilder debugContainerBuilder) : base(world)
+            IDebugContainerBuilder debugContainerBuilder,
+            WalkedDistanceAnalytics walkedDistanceAnalytics) : base(world)
         {
             this.analytics = analytics;
             this.realmData = realmData;
             this.playerEntity = playerEntity;
             this.identityCache = identityCache;
+            this.walkedDistanceAnalytics = walkedDistanceAnalytics;
 
-            currentIdentity = identityCache.Identity;
+            currentIdentity = identityCache?.Identity;
 
-            totalElevationGainBinding = new ElementBinding<string>(string.Empty);
             debugContainerBuilder
                .TryAddWidget("Badges Tracking")?
-               .AddCustomMarker("Vertical Voyager: ", totalElevationGainBinding);
+               .AddCustomMarker("Vertical Voyager: ", totalElevationGainBinding)
+               .AddCustomMarker("Steps Count: ", stepsCountBinding);
```

Comment:
Maybe, as we did for the "Vertical Voyager" badge, we should put here the official name of the badge related to the steps count to know what specific badge we're tracking?

```suggestion
               .AddCustomMarker("Walkabout Wanderer: ", stepsCountBinding);
```

Not an important thing, but let's agree a format 👍

Reply:
even for Voyager I would put what it is - "Elevation gain [m]". Since it is not clear what logic is applied to create badge out of this metric, its rules of calculation. For example, Walkabout Wanderer will perform on backend and sum all received "steps-count" analytics events. Moreover, since steps count sent each 30 sec and reset to zero, you will see "Walkabout Wanderer:" going 0... 15... 56.. 0.. 20... 70... 0..  Does it give any evidence about the badge? from my opinion - no. But it shows that the tracker works :)

Reply:
added both :)